vite.config.mts

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from 'vite';
+import react from '@vitejs/plugin-react';
+
+import vitePortScanner from './vite-plugin-port-scanner';
+
+// https://vite.dev/config/
+export default defineConfig({
+  plugins: [react(), vitePortScanner()]
```

Comment:
I think it would be nice to open the index page when running the dev server from the command line. We can also avoid passing the port to the dev server in the package.json with the following config:

```
server: {
    port: 9000,
    open: true
  },
  plugins: [react(), vitePortScanner()]
```

Reply:
Good tip regarding the port.
I don't think the `open` flag is very useful, since it opens on chrome by default, and it is not straightforward to set it to the default system browser. Given that people would need to configure this, it is equally easy to set that `open` flag.

Usually I just `cmd+click` the url and it opens the browser.

Reply:
@danielfdsilva I didn't notice it only opens with Chrome. I agree it is better to leave this config out.